notes/2024-02-06 biglep feedback for PR #65.md

```diff
@@ -0,0 +1,99 @@
+I wish github-mgmt didn’t sort all key alphabetically when there is a more meaningful default.  
+For example, with "collaborators” and “teams”, I want the keys be in descencing order: admin > maintain > push > triage > pull.
+Similarly, I want crucial properties like "archived" to be at the top.
+I think the default otherwise can be alphabetical.
+(Implementation wise I assume we do a custom sort function that pulls out any specific keys, sorts by that specified order, puts those first, and then adds the rest of the keys alphabetically.)
+
+# Somewhere between general and IPLD-specific
+These are at least things that apply to the 2024Q1 cleanup for ipld, libp2p, and ipfs orgs. Some of them can be generalized.
+
+## Archived repos cause clutter
+These .yaml files can be unwieldy.  One thing I think could help would be to have a view on all the repos that aren't archived.  Some ideas (I assume from least to most amount of work):
+* Put all the archived repos at the bottom of the file.  (I still don't like this one as much because archived repos will show up when doing text search for "team X" when trying to answer "what teams does X have permissions to").
+* Have an optional separate .yaml that merges in at build/process time.  This would allow a dedicated separate file (e.g., archived_repos.yaml).
+  * Generalizing this, maybe github-mgmt can be configured to do a merge of any specified files, which lets repo organizers decide if/how they want to break up their yaml file in general?
+* Go the IPFS route and actually have an "ipfs-inactive" org.  Archived projects can get moved there.  This makes things very clear about the maintenance status and also solves the declutter problem. 
+
+## Strip out access permissions to archived repos
+It adds clutter having individuals and teams showing up with access permissions in archived repos.  
+If a repo is archived, I think we should strip permissions.  
+Someone can always unarchive and add permissions through github-mgmt.  
+In addition to this reducing clutter, I this proposed process is good because it gives clear visibility to a significant repo event (e.g., unarchiving).
+
+## Remove the w3dt-stewards team
+That is legacy both in terms of name and function.
+It was used when there was a small team who was charged with looking after the whole github org and they didn't have github-mgmt at their disposal.
+In PLv10 / PL Innovation Network era, there is no corrolary for this.
+If broad action is needed, it can be done broadly via github-mgmt.  And where it can't be done directly with github-mgmt, github-mgmt can at least be done to give broad permissions to go do the broad action.
+
+## Default access of the ipdx team
+It looks like the ipdx team has admin permissions on most repos.   
+Should we maybe reduce this and instead ensure they just have github-mgmt admin access, which still enables them to "break glass" and escalate their access when needed?
+
+## Commit/PR order
```

Comment:
2024-02-08 conversation: agreed that it's good to have a commit/PR that is not cluttered so that when folks are @mentioned that they only see the changes that really pertain to them.